setup-gams/script.py

```diff
@@ -78,17 +78,19 @@ def get_info(uname, version: str) -> Tuple[str, str]:
         or install_dir.joinpath("gams.exe").is_file()
     ):
         pass  # Already exists, e.g. restored from cache → skip install
-    elif uname.system == "Windows":
+    else:
```

Comment:
```suggestion
    elif: uname.system == "Windows":
        pass
    else:
```

We could add this to prevent the warnings mentioned in https://github.com/iiasa/ixmp/pull/552#issuecomment-2820908257.